<a href="https://colab.research.google.com/github/routb68/hdlc/blob/main/HDLC_corpus.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import json

In [25]:
with open('/content/drive/MyDrive/sample data HDLC /model_data/specific_districts/train_split_44_districts.json', 'r') as f:
  data = json.load(f)

In [3]:
print(type(data))

<class 'dict'>


In [26]:
len(data)

124341

In [27]:
x = 0
for k in data:
  print(k)
  x+=1;
  if(x>10):
    break;

0
1
2
3
4
5
6
7
8
9
10


In [28]:
print(type(data['119481']))

<class 'dict'>


In [6]:
for k in data['119481']:
  print(k)

decision
segments
district
case_number


In [8]:
print(type(data['119481']['decision']))
print(type(data['119481']['segments']))
print(type(data['119481']['district']))
print(type(data['119481']['case_number']))

<class 'str'>
<class 'dict'>
<class 'str'>
<class 'str'>


In [10]:
print(data['119481']['decision'])
print(" ")
print(data['119481']['district'])
print(" ")
print(data['119481']['case_number'])

dismissed
 
bareilly
 
Bail_4702232_201919-06-20191907.txt


In [11]:
for k in data['119481']['segments']:
  print(k)

judge-opinion
facts-and-arguments


In [12]:
for k in data['119481']['segments']['facts-and-arguments']:
  print(k)

समर्थन में सगीर अहमद का शपथपत्र प्रस्तुतकिया गया है। 
अभियोजन कथानक के अनुसार दिनांक 24.03.2019 को वादिनीरुचि <नाम> द्वारा थाना नवाबगंज <नाम> इस आशय का मुकदमा कायम कराया कि दोवर्षं पूर्व प्रार्थिनी के मोबाइल संख्या <फ़ोन-नंबर> <नाम> काल आई और काल करनेवाले ने अपन नाम <नाम> बताया और प्रार्थिनी के <नाम> करने के <नाम> भी फोन करतारहा और प्रार्थिनी से बात चलती रही क्योंकि मुल्जिम ने शादी का झांसा <नाम> करनवाबगंज बुलाया और <नाम> पास स्थित कमरे <नाम> ले गया तथा बलात्कारकिया।  यह सिलसिला चलता रहा प्रार्थिनी ने जब शादी का आग्रह किया तो उक्तमुल्जिम प्रार्थिनी को दिनांक 02.02.2019 को अपने घर ले गया तो प्रार्थिनी को पतालगा कि वह मुस्लिम है तो प्रार्थिनी को अपने साथ धोखे का एहसास हुआ।  विरोधकरने <नाम> मुल्जिम व परिवार वालों ने मारा पीटा तथा कमरे में बंद <नाम> औरमुल्जिम के <नाम> उस्मान तथा छोटा <नाम> ने जबरन दुराचार किया तथा सभीमुल्जिमान ने <नाम> में जबरन गाय का मीट खिलाया और ज्यादा देरी होने <नाम> बहनने फोन किया तो उक्त मुल्जिमान ने फोन रिसीव किया और फोन <नाम> कहा कितुम्हारी बहन को हमने मुसलमान बना 

समर्थन में सगीर अहमद का शपथपत्र प्रस्तुतकिया गया है। 
अभियोजन कथानक के अनुसार दिनांक 24.03.2019 को वादिनीरुचि <नाम> द्वारा थाना नवाबगंज <नाम> इस आशय का मुकदमा कायम कराया कि दोवर्षं पूर्व प्रार्थिनी के मोबाइल संख्या <फ़ोन-नंबर> <नाम> काल आई और काल करनेवाले ने अपन नाम <नाम> बताया और प्रार्थिनी के <नाम> करने के <नाम> भी फोन करतारहा और प्रार्थिनी से बात चलती रही क्योंकि मुल्जिम ने शादी का झांसा <नाम> करनवाबगंज बुलाया और <नाम> पास स्थित कमरे <नाम> ले गया तथा बलात्कारकिया।  यह सिलसिला चलता रहा प्रार्थिनी ने जब शादी का आग्रह किया तो उक्तमुल्जिम प्रार्थिनी को दिनांक 02.02.2019 को अपने घर ले गया तो प्रार्थिनी को पतालगा कि वह मुस्लिम है तो प्रार्थिनी को अपने साथ धोखे का एहसास हुआ।  विरोधकरने <नाम> मुल्जिम व परिवार वालों ने मारा पीटा तथा कमरे में बंद <नाम> औरमुल्जिम के <नाम> उस्मान तथा छोटा <नाम> ने जबरन दुराचार किया तथा सभीमुल्जिमान ने <नाम> में जबरन गाय का मीट खिलाया और ज्यादा देरी होने <नाम> बहनने फोन किया तो उक्त मुल्जिमान ने फोन रिसीव किया और फोन <नाम> कहा कितुम्हारी बहन को हमने मुसलमान बना <नाम> है।  हम निकाह पढंगे तभी बहन ने100 नम्बर डायल को फोन किया जिल्होंने प्रार्थिनी को बचाया ।  प्रार्थिनी को वहां परपहुंच <नाम> पूर्व परिचित गांव का सुरेंद्र <नाम> अपने घर ले गया और प्रार्थिनी के मांबाप का सूचना दी।  इस बीच प्रार्थिनी को उक्त मुल्जिमान सुरेंद्र <नाम> एवंमुल्जिमान के परिवार वालों ने एक माह मे शादी कराने का आश्‍वासन <नाम> लेकिनकल दिनाक 23.03.2019 को शादी करने से साफ इंकार <नाम> <नाम> और <नाम> सेमारने की धमकी दी।  प्रार्थिनी को उक्त इरफान व परिवार एवं सुरेंद्र से <नाम> मालका खतरा है। 
अभियुक्त की ओर से प्रस्तुत जमानत प्रार्थनापत्र में कथन किया गयाहै कि उसको मिथ्या आलिप्त किया गया है।  अभियोजन कथानक बनावटी वमनगढंत है।  पीडिता की जन्मतिथि 01.05.1999 है और वह बालिग है।  <नाम> 164दं0प्रoसं0 के बयान में पीडिता ने घटना की कोई <नाम> एवं समय नहीं <नाम> है। पिछले दो वर्ष के दौरान अभियुक्त आर्मी पर्सनल होने के <नाम> ड्यूटी <नाम> रहा है अतः अभियोजन कथानक झूठा है।  वादिनी द्वारा कई बार मोबाइल कालअभियुक्त द्वारा करने का जिक हैं, परंतु उसके द्वारा <नाम> तो कहीँ शिकायत की गयीऔर <नाम> ही सिम ब्लॉक किया।  सहअभियुक्तगण के खिलाफ आरोप सही नहीं पायेगये इसलिये विवेचना में उनके नाम निकाल दिये गये।  उक्त आधारों <नाम> जमानतपर <नाम> किये जाने की <नाम> की गई हैं। 
सहायक जिला शासकीय अधिवक्ता फौजदारी तथा वादी के निजीविद्वान अधिवक्ता के द्वारा <नाम> किया गया है कि अभियुक्त ने वादिनी को पहले तोफोन <नाम> धोखे से <नाम> नाम से काल किया और फिर अपने घर बुला <नाम> शादीका झांसा <नाम> <नाम> कमरे में बंद <नाम> के उसकी इच्छा के विपरीत उसको कमरे में बंदकर के जबरदस्ती बलात्कार किया तथा अभियुक्त के विरोध <नाम> मारपीट व <नाम> सेमारने की धमकी दी गयी।  उसके परिवार वालों ने पीडिता को गाय का मीट भीखिलवाया तथा पीडिता के साथ दुराचार भी किया और अभियुक्त ने पीडिता सेशादी के आश्वासन के बावजूद दिनांक 02.02.19 को शादी करने से इनकार करदिया गया।  जमानत प्रार्थनापत्र निरस्त करने की <नाम> की गयी है। 

In [13]:
for k in data['119481']['segments']['judge-opinion']:
  print(k)

जमानत प्र्ार्थनापत्र <नाम> प्रार्थी / अभियुक्त के विद्वान अधिवक्ता तथाअभियोजन की ओर से विद्वान सहायक जिला शासकीय अधिवक्ता फौजदारी केतथा वादी की ओर से उपस्थित निजी विद्वान अधिवक्ता के तर्को को सुना एवंपुलिस प्रपत्रो का सम्यक अवलोकन किया । 
अभियोजन द्वारा प्रस्तुत केस डायरी के अवलोकन से विदित है किपीडिता को अभियुक्त द्वारा अपना नाम <नाम> बताकर फोन किया और नवाबगंजशादी का झांसा <नाम> <नाम> बुलाया और कमरे में बंद <नाम> के उसकी इच्छा के विपरीतजबरदस्ती बलात्कार किया गया।  वह पीडिता को अपने गांव रिछोला ले गया। पीडिता को अभियुक्त के मुस्लिम होने की जानकारी हुई।  पीडिता द्वारा विरोध करनेपर उसको अभियुक्त तथा उसके परिवार वालों द्वारा मारा पीटा गया।  पीडिता द्वाराअभियुक्त से शादी के लिये कहने <नाम> उसको मारा पीटा गया।  अभियुक्त के भाईउस्मान तथा छोटा <नाम> ने जबरन दुराचार किया व गाय का मीट खिलाया।  दिनांक23.03.2019 को अभियुक्त द्वारा पीडिता से शादी करने से साफ इनकार <नाम> दियागया।  पीडिता ने अपने बयान <नाम> 161 द॑0प्रoसं0 में अभियोजन कथानक कासमर्थन किया है और कहा है कि दो वर्ष पूर्व फोन <नाम> अपना न

जमानत प्र्ार्थनापत्र <नाम> प्रार्थी / अभियुक्त के विद्वान अधिवक्ता तथाअभियोजन की ओर से विद्वान सहायक जिला शासकीय अधिवक्ता फौजदारी केतथा वादी की ओर से उपस्थित निजी विद्वान अधिवक्ता के तर्को को सुना एवंपुलिस प्रपत्रो का सम्यक अवलोकन किया । 
अभियोजन द्वारा प्रस्तुत केस डायरी के अवलोकन से विदित है किपीडिता को अभियुक्त द्वारा अपना नाम <नाम> बताकर फोन किया और नवाबगंजशादी का झांसा <नाम> <नाम> बुलाया और कमरे में बंद <नाम> के उसकी इच्छा के विपरीतजबरदस्ती बलात्कार किया गया।  वह पीडिता को अपने गांव रिछोला ले गया। पीडिता को अभियुक्त के मुस्लिम होने की जानकारी हुई।  पीडिता द्वारा विरोध करनेपर उसको अभियुक्त तथा उसके परिवार वालों द्वारा मारा पीटा गया।  पीडिता द्वाराअभियुक्त से शादी के लिये कहने <नाम> उसको मारा पीटा गया।  अभियुक्त के भाईउस्मान तथा छोटा <नाम> ने जबरन दुराचार किया व गाय का मीट खिलाया।  दिनांक23.03.2019 को अभियुक्त द्वारा पीडिता से शादी करने से साफ इनकार <नाम> दियागया।  पीडिता ने अपने बयान <नाम> 161 द॑0प्रoसं0 में अभियोजन कथानक कासमर्थन किया है और कहा है कि दो वर्ष पूर्व फोन <नाम> अपना नाम <नाम> बतायामना करने <नाम> भी बात करता रहा शादी का झांसा <नाम> <नाम> नवाबगंज बुला करबाईपास स्थित कमरे <नाम> ले जा <नाम> बुरा <नाम> किया।  आर्मी में नौकरी करता हैजब छुटटी आता तो कमरे <नाम> बुला लेता था।  दिनांक 02.02.18 के <नाम> अपने घररिछोला अभियुक्त बुला ले गया।  तब पता चला कि अभियुक्त मुसलमान है।  उसनेव उसके घर वालों ने कमरे में बंद <नाम> के उसके साथ मारपीट की, गांय का मासखिलवाया इसका सही नाम इरफान है।  पीडिता ने अपने बयान <नाम> 164 दं0प्र०सं0में भी अपने बयान <नाम> 161 दंएप्र0सं0 का समर्थन किया है।  पीडिता के शैक्षिकप्रमाणपत्र के अनुसार उसकी जन्मतिथि दिनांक 07.05.2002 है जिसके आधार परपीडिता घटना के दिनांक को अवयस्क थी। 

In [14]:
with open('/content/drive/MyDrive/sample data HDLC /model_data/all_districts/train_split_alldistricts.json', 'r') as f:
  data = json.load(f)

In [17]:
print(type(data))
len(data)

<class 'dict'>


123742

In [18]:
a = 0
for k in data:
  print(k)
  a+=1;
  if(a>10):
    break;

0
1
2
3
4
5
6
7
8
9
10


In [19]:
for k in data['1']:
  print(k)

decision
segments
district
case_number


In [20]:
print(type(data['1']['decision']))
print(type(data['1']['segments']))
print(type(data['1']['district']))
print(type(data['1']['case_number']))

<class 'str'>
<class 'dict'>
<class 'str'>
<class 'str'>


In [21]:
print(data['1']['decision'])
print(" ")
print(data['1']['district'])
print(" ")
print(data['1']['case_number'])

granted
 
agra
 
Bail Application_1081_202111-02-20212541.txt


In [22]:
for k in data['1']['segments']:
  print(k)

judge-opinion
facts-and-arguments


In [23]:
for k in data['1']['segments']['facts-and-arguments']:
  print(k)

इस जमानत प्रार्थना पत्र के साथ आवेदक / अभियुक्त क द्वारास्वयं का शपथ पत्र प्रस्तुत किया गया है। 
अभियोजन कथानक के अनुसार वादी मुकदमा <नाम> कुमारअवर अभियन्ता 33/11 के.वी.उपकन्द्र, जिला <नाम> द्वारा थाना जगदीशपुरा,आगरा <नाम> इस आशय की प्रथम सूचना रिपोर्ट दर्ज करायी कि - दिनांक13.05.2018,// 14.05.2018 की <नाम> के बीच टीम के अन्य सदस्यों के साथचैकिंग किये जाने <नाम> अभियुक्ता <नाम> सरोज को अवैध <नाम> से विद्युत काउपयोग <नाम> चोरी करते हुए पाया गया।  इस संबंध में प्राथमिकी दर्ज कीगयी । 
्रार्थी/अभियुक्ता के विद्वान अधिवक्ता द्वारा तर्क रख गया हैकि प्रार्थी/अभियुक्ता ने कोई अपराध नहीं किया है, उसे इस कस में झूंठाफॅसाया गया है।  प््रार्थी/अभियुक्त के द्वारा विद्युत विभाग का शमनशुल्क / बकाया जमा <नाम> <नाम> गया है।  प्रार्थी/ अभियुक्ता न्यायिक अभिरक्षा मेंहै।  अत: जमानत प्रार्थना पत्र स्वीकार किया जाय। 


इस जमानत प्रार्थना पत्र के साथ आवेदक / अभियुक्त क द्वारास्वयं का शपथ पत्र प्रस्तुत किया गया है। 
अभियोजन कथानक के अनुसार वादी मुकदमा <नाम> कुमारअवर अभियन्ता 33/11 के.वी.उपकन्द्र, जिला <नाम> द्वारा थाना जगदीशपुरा,आगरा <नाम> इस आशय की प्रथम सूचना रिपोर्ट दर्ज करायी कि - दिनांक13.05.2018,// 14.05.2018 की <नाम> के बीच टीम के अन्य सदस्यों के साथचैकिंग किये जाने <नाम> अभियुक्ता <नाम> सरोज को अवैध <नाम> से विद्युत काउपयोग <नाम> चोरी करते हुए पाया गया।  इस संबंध में प्राथमिकी दर्ज कीगयी । 
्रार्थी/अभियुक्ता के विद्वान अधिवक्ता द्वारा तर्क रख गया हैकि प्रार्थी/अभियुक्ता ने कोई अपराध नहीं किया है, उसे इस कस में झूंठाफॅसाया गया है।  प््रार्थी/अभियुक्त के द्वारा विद्युत विभाग का शमनशुल्क / बकाया जमा <नाम> <नाम> गया है।  प्रार्थी/ अभियुक्ता न्यायिक अभिरक्षा मेंहै।  अत: जमानत प्रार्थना पत्र स्वीकार किया जाय। 

In [24]:
for k in data['1']['segments']['judge-opinion']:
  print(k)

 (2)उभय पक्षों के विद्धान अधिवक्तागण के तर्को को सुना तथापत्रावली का अवलोकन किया। 
विद्युत विभाग क विद्वान अधिवक्ता द्वारा कहा गया है कि्रार्शी/ अभियुक्त के द्वारा शमन शुल्क जमा कराया जा चुका है, आपत्ति नहींहै। प्रार्थी / अभियुक्त की ओर से विद्युत विभाग की रसीद संख्या-ए772706 / 28 दिनांकित 01.06.2018 अंकन 4000/- रूपये दाखिल की गयीहै।  इस संबंध में कार्यालय अधिशासी अभियन्ता विद्युत वितरण खण्ड, कमलानगर, जिला <नाम> के द्वारा थानाध्यक्ष जगदीशपुरा, जनपद <नाम> को एकपत्र इस आशय का प्रेषित किया गया है कि उपभोक्ता द्वारा अपना शमनशुल्क जमा <नाम> <नाम> है।  अतः शमन के आधार <नाम> पंजीकृत अपराध कोसमाप्त करने का कष्ट करें।  इस पत्र <नाम> विद्युत विभाग के विद्वानअधिवक्ता द्वारा सब्मिट का पृष्ठांकन किया गया है। 


 (2)उभय पक्षों के विद्धान अधिवक्तागण के तर्को को सुना तथापत्रावली का अवलोकन किया। 
विद्युत विभाग क विद्वान अधिवक्ता द्वारा कहा गया है कि्रार्शी/ अभियुक्त के द्वारा शमन शुल्क जमा कराया जा चुका है, आपत्ति नहींहै। प्रार्थी / अभियुक्त की ओर से विद्युत विभाग की रसीद संख्या-ए772706 / 28 दिनांकित 01.06.2018 अंकन 4000/- रूपये दाखिल की गयीहै।  इस संबंध में कार्यालय अधिशासी अभियन्ता विद्युत वितरण खण्ड, कमलानगर, जिला <नाम> के द्वारा थानाध्यक्ष जगदीशपुरा, जनपद <नाम> को एकपत्र इस आशय का प्रेषित किया गया है कि उपभोक्ता द्वारा अपना शमनशुल्क जमा <नाम> <नाम> है।  अतः शमन के आधार <नाम> पंजीकृत अपराध कोसमाप्त करने का कष्ट करें।  इस पत्र <नाम> विद्युत विभाग के विद्वानअधिवक्ता द्वारा सब्मिट का पृष्ठांकन किया गया है। 